ethcore/sync/src/chain/requester.rs

```diff
@@ -0,0 +1,154 @@
+// Copyright 2015-2018 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+use api::{WARP_SYNC_PROTOCOL_ID};
+use block_sync::{BlockRequest};
+use bytes::Bytes;
+use ethcore::header::{BlockNumber};
+use ethereum_types::{H256};
+use network::{PeerId, PacketId};
+use rlp::{RlpStream};
+use std::time::{Instant};
+use sync_io::SyncIo;
+
+use super::{
+	BlockSet,
+	ChainSync,
+	PeerAsking,
+
+	ETH_PACKET_COUNT,
+	GET_BLOCK_BODIES_PACKET,
+	GET_BLOCK_HEADERS_PACKET,
+	GET_RECEIPTS_PACKET,
+	GET_SNAPSHOT_DATA_PACKET,
+	GET_SNAPSHOT_MANIFEST_PACKET,
+};
+
+pub struct SyncRequester {}
```

Comment:
Docs

Reply:
and just `pub struct SyncRequester;`, no braces